ui/components/MesheryPerformance/Dashboard.js

```diff
@@ -159,7 +158,13 @@ function Dashboard({ updateProgress, enqueueSnackbar, closeSnackbar, grafana, ro
           </Paper>
         </Grid>
       </Grid>
+      <Grid>
+        <Paper className={classes.paper}>
+      <PerformanceCalendar style={{ height: "40rem", margin: "2rem 0 0" }} />

```

Comment:
This height will vary, so, better not to keep it fixed ✌🏽

Reply:
This was there from before itself, i didnt make any changes, if we dont specify a height the calendar collapses into a line.

Reply:
Oh ,ok, so, instead of height, we can change it to `min-height` ig

Reply:
Ive tried doing that, but then the grid of the calendar goes away😬.

Reply:
Ok, it will not create any issues as of now, so, we can stick with this 👍🏽